spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -20,110 +20,126 @@ On Ethereum, there are two types of accounts: externally owned accounts, control
 
 ## Technical Specification
 
-The implementation of interchain account is non-symmetric. This means that each chain can have a different way to generate an interchain account and deserialise the transaction bytes and a different set of transactions that they can execute. For example, chains that use the Cosmos SDK will deserialise tx bytes using Amino or Protobuf, but if the counterparty chain is a smart contract on Ethereum, it may deserialise tx bytes by an ABI that is a minimal serialisation algorithm for the smart contract.
-The interchain account specification defines the general way to register an interchain account and transfer tx bytes. The counterparty chain is responsible for deserialising and executing the tx bytes, and the sending chain should know how counterparty chain will handle the tx bytes in advance.
+The implementation of interchain accounts is non-symmetric. This means that each sending chain can have a different way to generate an interchain account and each receiving chain a different way to deserialise the transaction bytes and a different set of transactions that they can execute. For example, chains that use the Cosmos SDK will deserialise tx bytes using Protobuf, but if the counterparty chain is a smart contract on Ethereum, it may deserialise tx bytes by an ABI that is a minimal serialisation algorithm for the smart contract.
+
+A chain can implement one or both parts to the interchain accounts protocol (sending and receiving). A sending chain registering and controlling an account does not necessarily have to allow other chains to register accounts on its own chain, and vice versa. 
 
-Each chain must satisfy following features to create a interchain account:
+The interchain account specification defines the general way to register an interchain account and transfer tx bytes. The counterparty chain is responsible for deserialising and executing the tx bytes, and the sending chain should know how the counterparty chain will handle the tx bytes in advance. Each chain-specific implementation should clearly document how serialization/deserialization of transactions happens and ensure the required packet data format is clearly outlined. 
 
-- New interchain accounts must not conflict with existing ones.
-- Each chain must keep track of which counterparty chain created each new interchain account.
+Each chain must satisfy the following features to create an interchain account:
 
-Also, each chain must know how the counterparty chains serialise/deserialise transaction bytes in order to send transactions via IBC. And the counterparty chain must implement the process of safely executing IBC transactions by verifying the authority of the transaction's signers.
+- New interchain accounts must not conflict with existing ones
+- Each chain must keep track of which counterparty chain created each new interchain account
```

Comment:
Isn't this implicit via IBC channels?

Reply:
What do you mean by "via IBC channels"? 

I think this is quite implicit in general, you're correct. I think it's okay to leave it explicit though.

Reply:
Ahh, I think it is supposed to convey that each account needs to be associated with a channel?

Reply:
I think the point here is that the sending chain needs to keep track of which accounts it has registered and on what chain. Likewise, the receiving side needs to do the same. 

Clearly, this should be communicated better. The line in question was added in the previous version of the spec, and I may have interrupted it differently than you have.

Reply:
Yea, I think given the context of IBC, it is more concise to specify that the sending/receivers must track which channels are used for which accounts. IBC doesn't really have a strong notion of "chain", rather it has clients. Connections built on clients also have security implications. Thus two accounts using the different channels but the same clients might have completely different security guarantees